app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,23 @@
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+ 
+  def perform(initiator, benchmark_type, benchmark_result_type, category)
+    commit_objects = BenchmarkRun.where(benchmark_type: benchmark_type,
+                     benchmark_result_type: benchmark_result_type).offset(10).limit(1000)
```

Comment:
> Since, we are fetching 1000 results it doesn't make much of a difference if we leave a first few results.

It does make a difference if the last 10 results are way off. What you want here is the benchmark results of  the last 1000 commits which you can easily fetch.